packages/admin/admin/src/fullPageAlert/FullPageAlert.tsx

```diff
@@ -122,7 +122,7 @@ export const FullPageAlert: FunctionComponent<FullPageAlertProps> = (inProps) =>
             <ContentContainer {...slotProps.contentContainer}>
                 <InlineAlertStyled
                     title={
-                        <Title variant="h3" {...slotProps.title}>
+                        <Title variant="h3" component="span" {...slotProps.title}>
```

Comment:
Wouldn't it be better to override the InlineAlert's title variant to h3 using slotProps? Now it's a h3 in a h5.

Reply:
yeah agree, slots should be used for this 👍